prime-router/src/main/kotlin/cli/tests/CompareData.kt

```diff
@@ -741,7 +741,7 @@ class CompareCsvData {
                                 )
                                 passed = false
                             }
-                        } catch (e: DateTimeParseException) {
+                        } catch (e: Throwable) {
```

Comment:
Hmm. Why go from a narrow to an everything handler? It is generally considered a bad practice because it could mask something.